server/routes/lecture-route.js

```diff
@@ -1,12 +1,14 @@
 const express = require("express");
 
 // Import lecture-controller
-const lecturebookingRoutes = require("./../controllers/lecture-controller");
+const lectureRoutes = require("./../controllers/lecture-controller");
 
 // Create router
 const router = express.Router();
 
-router.get("/newbooking", lecturebookingRoutes.newBooking);
+router.get("/bookable/:userId", lectureRoutes.getBookingLectures);
+router.get("/:lectureId/book", lectureRoutes.newBooking);
```

Comment:
It would be better to use POST instead of GET to insert a new booking. Usually GET is used only to access resources and not to create new ones

Reply:
Sure, just a copy-paste error